src/components/search-form.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+class SearchForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchText: '',
+    };
+    this.handleInputChange = this.handleInputChange.bind(this);
```

Comment:
You wouldn't need to do this line if you set your methods as arrow functions. For example, your `handleInputChange(event) {}`,  would become `handleInputChange = (event) => {}` and it would automatically bind it's context.